germanium/decorators.py

```diff
@@ -142,12 +142,15 @@ def call_test_method(method, self, data, named_data, use_rollback=False):
     elif named_data and not isinstance(data, NamedTestData):
         named_data = None
 
+    is_data_consumer = getattr(method, 'is_data_consumer', False)
     if use_rollback:
         sid = transaction.savepoint()
     try:
-        if getattr(method, 'is_data_consumer', False):
+        if is_data_consumer:
             method(self, data=data, named_data=named_data)
         else:
+            if hasattr(self, 'tear_up_data_consumer'):
```

Comment:
shouldn't the method be set_up_data_consumer?

Reply:
it is called according to unit tests